lambda/fetchDeviceShadow.ts

```diff
@@ -215,7 +215,9 @@ const h = async (): Promise<void> => {
 					'shadowAge',
 					MetricUnits.Seconds,
 					Math.round(Date.now() / 1000) -
-						getShadowUpdateTime(deviceShadow.state.metadata),
+						getShadowUpdateTime(
+							deviceShadow.state.metadata as Record<string, any>,
```

Comment:
I need to cast `deviceShadow.state.metadata` to `Record<string, any>` because `getShadowUpdateTime` requires

Reply:
Yeah, recursive typing is not really possible with TypeBox (and JSON schema)